css/logo-nav.css

```diff
@@ -56,100 +56,250 @@ animation-delay: 0.7s;
+   .nav>li>a {
+       font-size: 14px;
+       font-weight: 400;
+   }
+   .nav a {
+       width: 110px;
+   }
+   .dropdown-menu > li > a {
+        font-size: 13px;
+    }
 }
-@media(min-width:768px) {
+@media(max-width:992px)  {
 
-    .navbar-fixed-top .navbar-brand {
-        padding: 15px 0;
+    .navbar-collapse.collapse {
+        display: none !important;
     }
-    .tab {
-        text-align: center;
+
+    .nav>li>a {
+        font-size: 13px;
+        letter-spacing: 1px;
     }
-}
-@media(max-width:992px) {
-    .language-set #feature {
-        display: block;
+
+    .nav > li > a:hover {
+       background-color: #E7E7E7;
+       color: black;
     }
-    .notebook-preview img {
-        margin: 0 auto;
+
+    .nav > li > a:focus {
+        background-color: #E7E7E7 !important;
     }
-    #notebookicon {
+    .dropdown-menu>li>a {
+        font-size: 13px;
+        letter-spacing: 1px;
+    }
+    .dropdown-menu > li  {
+       background-color: #f6f6f6;
+    }
+    .dropdown-menu {
+        display: none !important;
+    }
+    .dropdown  a  {
+        text-align: left !important;
+    }
+    .dropdown  a  {
+        padding: 10px 15px 10px 15px !important;
+    }
+    .dropdown {
+        padding-left: 25px;
+    }
+    .navbar-toggle {
+        margin: 23px 0 23px 0;
+        padding: 9px 0 9px 0;
+    }
+    .navbar-toggle {
         display: block;
-        margin: 0 auto;
+        margin-left: 15px;
+    }
+    .navbar-header {
+        float: none;
+    }
+    .navbar-collapse {
+        border-top: 1px solid transparent;
+        box-shadow: inset 0 1px 0 rgba(255,255,255,0.1);
+    }
+    .navbar-fixed-top {
+        top: 0;
+        border-width: 0 0 1px;
+    }
+    .navbar-nav {
+        float: none!important;
+        margin-top: 7.5px;
+    }
+    .navbar-nav>li {
+        float: none;
+    }
+    .navbar-nav>li>a {
+        padding-top: 10px;
         padding-bottom: 10px;
     }
-    .notebook-preview h1 {
-        text-align: center;
+    .nav a {
+        width: auto;
+    }
+   .navbar-nav .open .dropdown-menu {
+       position: static;
+       float: none;
+       width: auto;
+       background-color: transparent;
+       border: 0;
+       -webkit-box-shadow: none;
+       box-shadow: none;
+    } 
+    .nav:first-child {
+        border-top: 2px solid #F2F2F2;
+    }
+    .nav .open>a, .nav .open>a:focus, .nav .open>a:hover {
+        border-color: #F2F2F2;
+    }
+   .navbar-scroll .navbar-inner .dropdown-menu > :first-child {
+        border-top: none;
+    }
+    .open .dropdown-menu {
+        display: block !important;
+    }
+    .collapse.in{
+        display:block !important;
+    }
+    #maintabs a {
+        text-align: left !important;
+    }
+    #maintabs a {
+        padding: 10px 15px 10px 15px;
+    }
+    #carrot {
+        float: right;
+        padding-top: 8px;
+    }
+}
+@media (max-width: 768px) {
+    .navbar-header {
+        margin-left: 0px !important;
```

Comment:
in general, `0` should not have units. because `0px` == `0pt` == `0em`. I know our css was already using units, but let's try to make the code better. There is slight performance improvement in not including units, at the browser will be faster at parsing and converting it.